src/map/ai/controllers/mob_controller.cpp

```diff
@@ -1,4 +1,4 @@
-﻿/*
```

Comment:
Looks like the files BOM (byte order mark) got changed

Reply:
This happens constantly. I'm sure if one were to check them all, they would be pretty inconsistent